src/constants/enums.js

```diff
@@ -0,0 +1,38 @@
+export const RecordType = {
+  ALL: 'ALL',
```

Comment:
If we can ensure this is documented here - https://nhse-dsic.atlassian.net/wiki/spaces/TW/pages/12509020254/ORC+-+Databases - that would be great!